front/src/modules/ui/input/components/Checkbox.tsx

```diff
@@ -32,7 +32,7 @@ const StyledInput = styled.input<{ indeterminate?: boolean }>`
 
   & + label:before {
     background: ${({ theme }) => theme.background.primary};
-    border: 1px solid ${({ theme }) => theme.border.color.strong};
+    border: 1px solid ${({ theme }) => theme.font.color.secondary};
```

Comment:
we should not use font color. We should discuss with design and update border.inverted to be either #333 or #666